BUILD

```diff
@@ -0,0 +1,10 @@
+load("//tools/bzl:plugin.bzl", "gerrit_plugin")
```

Comment:
I'm ok with moving to Bazel build, but I'll need a short readme about how to do that. I tried by following how other gerrit plugins are build, but did not succeed (where does the //tools "rule" should come from?)